caffe2/operators/hip/conv_op_miopen.hip

```diff
@@ -499,47 +523,47 @@ bool MIOPENConvGradientOp::DoRunWithType() {
           bottom_desc_,
           &bwdDataWsSize_));
       if ((bwdDataWsSize_ > 0) && (bwdDataWs_ == nullptr)) {
-          HIP_CHECK(hipMalloc(&bwdDataWs_, bwdDataWsSize_));
+        HIP_CHECK(hipMalloc(&bwdDataWs_, bwdDataWsSize_));
       }
 
       miopen_wrapper_.with_miopen_state(miopen_state_, [&](MIOPENState* state) {
         MIOPEN_ENFORCE(miopenFindConvolutionBackwardDataAlgorithm(
-          state->miopen_handle(),
-          top_desc_,
-          dY.template data<T_DY>(),
-          weight_desc_,
-          Weight.template data<T_W>(),
-          conv_desc_,
-          bottom_desc_,
-          dX->template mutable_data<T_DX>(),
-          requestAlgoCount_,
-          &returnedAlgoCount_,
-          &perf,
-          bwdDataWs_,
-          bwdDataWsSize_,
-          false));
+            state->miopen_handle(),
+            top_desc_,
```

Comment:
is that just extra white space added?

Reply:
This is made by clang format for codestyle.

Reply:
@shahzadlone do you have any concern?

Reply:
@houseroad no, just wanted to confirm the author knew there was extra spacing being added, because sometimes our text editors can do some auto spacing without our knowledge.